package.json

```diff
@@ -5,7 +5,7 @@
   "main": "build/src/index.js",
   "scripts": {
     "install": "node-gyp rebuild",
-    "test": "true",
+    "test": "mocha build/test/test-*.js",
```

Comment:
Any specific reason for picking mocha?

Reply:
No -- just saw that trace and debug seemed to use mocha and decided to be consistent. 
I'd be happy to change if you'd recommend something else.

Reply:
mocha is what a lot of repos use indeed, and consistency is important. In gts itself we started using [ava](https://npm.im/ava) because it comes with good typescript and async function support out of the box. It also has some other nice features (parallel, better assertions).

However, at this point I don't think we have enough experience with ava to recommend it over mocha; but I would strongly encourage you to take a look at it.

One thing you may not necessarily like however, is that it runs tests in parallel. This means that your tests have to be non-interfering. While this seems a bit painful to get started with, I think this has longer term benefits in forcing healthier unit tests.

Reply:
This is more related to #12, but would if be possible still to change set value of process.env.GCLOUD_PROJECT (for example) in some tests and delete the value in other tests if I switched to ava?

I realize that running unit tests in parallel is ideal, but I also feel that it is very important to be able to modify the value of process.env.GCLOUD_PROJECT (and similarly used environment variables) in unit tests for initConfig().